cli.py

```diff
@@ -194,9 +194,9 @@ def get_gravatar_url(email, size):
                                 a.info['thumbnail'] = filename
                                 a.info['container'] = container
                                 db.session.commit()
-                                print "Done!"
+                                print("Done!")
                         except:
```

Comment:
- To catch any exception use the ``except Exception``, instead of 
  ``except BaseException`` or just ``except``. 
- Indicate the most specific exception type in ``except`` block. 

```python


     # Bad
     try:
         mapping[key]
     except Exception:
         ...

     # Better
     try:
         mapping[key]
     except KeyError:
         ...
```